frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -17,6 +17,10 @@ action.GitMachete.PushSelectedBranchAction.text=Push Selected Branch...
 action.GitMachete.PushSelectedBranchAction.GitMacheteContextMenu.text=_Push Branch...
 action.GitMachete.PushSelectedBranchAction.description=Push this branch using push dialog
 
+action.GitMachete.ResetSelectedBranchAction.text=Reset Selected Branch
```

Comment:
Also, I'd specify that it's about `reset to remote (tracking branch)` in every single user-facing message... just `reset branch` with no further context is very confusing